src/AddItem.js

```diff
@@ -3,28 +3,73 @@ import firebase from 'firebase';
 
 const AddItem = () => {
   const [inputValue, setInputValue] = useState();
+  const [frequency, setFrequency] = useState(7);
+  const [success, setSuccess] = useState(false);
 
   const addItem = event => {
     event.preventDefault();
+
     return firebase
       .firestore()
       .collection('items')
-      .add({ name: inputValue });
+      .add({
+        token: '143',
+        name: inputValue,
+        frequency: frequency,
+        lastPurchased: null,
+      })
+      .then(setSuccess(true));
+  };
+
+  const handleOnChange = e => {
+    setFrequency(parseInt(e.target.value));
   };
 
   return (
     <div>
       <form>
+        <label htmlFor="item">Item</label>
         <input
+          id="item"
           type="text"
           name="items"
           placeholder="enter grocery item here"
           onChange={event => setInputValue(event.target.value)}
         />
+        <fieldset>
+          <label htmlFor="soon">Soon</label>
+          <input
+            id="soon"
+            name="frequency"
+            type="radio"
+            value={7}
+            onChange={handleOnChange}
+            checked={frequency === 7}
+          />
+          <label htmlFor="kindofsoon">Kind of Soon</label>
+          <input
+            id="kindofsoon"
+            name="frequency"
+            type="radio"
+            value={14}
+            onChange={handleOnChange}
+            checked={frequency === 14}
+          />
+          <label htmlFor="notsoon">Not Soon</label>
+          <input
+            id="notsoon"
+            name="frequency"
+            type="radio"
+            value={30}
+            onChange={handleOnChange}
+            checked={frequency === 30}
+          />
+        </fieldset>
         <button type="submit" onClick={event => addItem(event)}>
           submit
         </button>
       </form>
+      {success ? 'Success' : null}
```

Comment:
I like this a lot, a lot!!! Would it be worthwhile to tell the user which item was added? Something like "Broccoli was added to your list." That's totally beyond AC . . .

Reply:
That would be very useful from a ux/ui perspective. Is it part of another AC? We tried to stick to the given AC since we didn't want to go off track/do too much. Totes can add it now if needed.

Reply:
There aren't any other tickets that address this directly. You could make a case for it being relevant to [#17](https://github.com/the-collab-lab/tcl-9-smart-shopping-list/issues/17). But you could also get that set up here as a nice addition.